react/Wrapper.tsx

```diff
@@ -16,6 +16,7 @@ interface Props {
   selectedSeller: Seller | undefined
   text?: string
   unavailableText?: string
+  goToProductPage: boolean | null
```

Comment:
We usually don't like `boolean` props, I think the best way to do this is to use a prop like:
```suggestion
  onClickBehavior?: 'add-to-cart' | 'go-to-product-page'
```

And the default would be `add-to-cart`

Reply:
@Klynger done, I already did.
Thank you